lib/tasks/github_usage_tracker.rake

```diff
@@ -0,0 +1,6 @@
+namespace :github_usage_tracker do
+  desc "Record a current datapoint of GitHub API usage"
+  task :record_datapoint => :environment do
+    GithubUsageTracker.record_datapoint
+  end
```

Comment:
Ok; I'm good with it if it can have a queue separate from the current ones. I just wanted to make sure it has a higher degree of likelihood that it wouldn't get stuck or displaced due to all the other things we queue up but this seems good. 👍